ref/feature/openidConnectServer/examples.adoc

```diff
@@ -8,7 +8,13 @@ Add the OP Open Liberty feature and any other needed features to the server.xml
 Define an OAuth service provider.
 Any user that is authorized to use OpenID Connect must also be mapped to the authenticated OAuth-role.
 
-In the following example, the OP expects the client’s SSL port to be set to 443:
+The following example assumes using basic registry which is available for purposes of development/test.
+If using another method of authentication, for example, LDAP, replace the `basicRegistry` section with link:https://www.openliberty.io/docs/ref/feature/#ldapRegistry-3.0.html[LDAP] config.
+
+The example uses `localStore` to store client data and token status, which means all client data and token status is held in memory, for test/development purposes.
+If you use a database, replace the `localStore` section with link:https://www.openliberty.io/docs/ref/feature/#jdbc-4.3.html[JDBC] database configuration.
+When using a database, clients are not specified in server.xml, instead they are added to the database through the registration endpoint.
```

Comment:
Change
"When using a database, clients are not specified in server.xml, instead they are added to the database through the registration endpoint."
To
"When you use a database, clients are not specified in the `server.xml` file. Instead, they are added to the database through the registration endpoint."